<launch>

  <node pkg="turtlesim" name="rost_turtle" type="turtlesim_node"/>

  <!-- PRODUCTION -->
  <node pkg="rostful" name="rost_server" type="gunicorn" args="">
    <param name="topics" value="['/turtle1/pose']" type="str" />
    <param name="services" value="['/clear','/reset','/kill','/spawn']" type="str" />
    <param name="actions" value="[]" type="str" />
  </node>



</launch>
